### PR TITLE
add parameter:'force_cpu' for some compare_op

### PR DESCRIPTION
**Pull request by Channingss**
add parameter:'force_cpu' for some compare_op：
-greater_equal
-less_equal
-greater_than
-equal
-not_equal

equal
greater_equal
greater_than
less_equal
not_equal

### CODE REVIEW

File: python/paddle/fluid/layers/control_flow.py [hunk 1]
```diff
@@ -1351,16 +1351,18 @@ def less_than(x, y, force_cpu=None, cond=None):
...
 
     Args:
-        x(Variable): First input to compare which is N-D tensor. The input data type should be float32, float64, int32, int64. 
+        x(Variable): First input to compare which is N-D tensor. The input data type should be float32, float64, int32, int64.
         y(Variable): Second input to compare which is N-D tensor. The input data type should be float32, float64, int32, int64.
         cond(Variable, optional): If is :attr:`None`, the op will create a variable as output tensor, the input shape and data type of \
             this tensor is the same as input :attr:`x`. If is not :attr:`None`, the op will set the variable as output tensor, the input shape \
             and data type of this tensor should be the same as input :attr:`x`. Default value is :attr:`None`.
+        force_cpu(bool, optional): Whether force to store the output tensor in CPU. If :attr:`force_cpu` is False,
+            the output tensor will be stored in running device memory. The defalut value is None, the output tensor will stored in the CPU memory.
```
Comment (Boyan-Liu) [inline python/paddle/fluid/layers/control_flow.py]: default拼写错误，
the output tensor will 'be' stored，少个'be'

File: python/paddle/fluid/layers/control_flow.py [hunk 2]
```diff
@@ -1395,16 +1401,18 @@ def less_equal(x, y, cond=None):
...
-
+    
     Args:
         x(Variable): First input to compare which is N-D tensor. The input data type should be float32, float64, int32, int64. 
         y(Variable): Second input to compare which is N-D tensor. The input data type should be float32, float64, int32, int64.
         cond(Variable, optional): If is :attr:`None`, the op will create a variable as output tensor, the shape and data type of this \
             tensor is the same as input :attr:`x` . If is not :attr:`None`, the op will set the variable as output tensor, the shape and data type \
             of this tensor should be the same as input :attr:`x` . Default value is :attr:`None`.
+        force_cpu(bool, optional): Whether force to store the output tensor in CPU. If :attr:`force_cpu` is False, 
+            the output tensor will be stored in running device memory. The defalut value is None, the output tensor will stored in the CPU memory.
```
Comment (Boyan-Liu) [inline python/paddle/fluid/layers/control_flow.py]: 语法错误同上

File: python/paddle/fluid/layers/control_flow.py [hunk 3]
```diff
@@ -1438,16 +1451,18 @@ def greater_than(x, y, cond=None):
...
-
+    
     Args:
         x(Variable): First input to compare which is N-D tensor. The input data type should be float32, float64, int32, int64. 
         y(Variable): Second input to compare which is N-D tensor. The input data type should be float32, float64, int32, int64.
         cond(Variable, optional): If is :attr:`None` , the op will create a variable as output tensor, the shape and data type of this \
             tensor is the same as input :attr:`x`. If is not :attr:`None` , the op will set the variable as output tensor, the shape and data \
             type of this tensor is the same as input :attr:`x`. Default value is :attr:`None`.
+        force_cpu(bool, optional): Whether force to store the output tensor in CPU. If :attr:`force_cpu` is False, 
+            the output tensor will be stored in running device memory. The defalut value is None, the output tensor will stored in the CPU memory.
```
Comment (Boyan-Liu) [inline python/paddle/fluid/layers/control_flow.py]: 语法错误同上

File: python/paddle/fluid/layers/control_flow.py [hunk 4]
```diff
@@ -1492,6 +1510,8 @@ def equal(x, y, cond=None):
         cond(Variable, optional): Optional output which can be any created 
             Variable that meets the requirements to store the result of *equal*.
             if cond is None, a new Varibale will be created to store the result.
+        force_cpu(bool, optional): Whether force to store the output tensor in CPU. If :attr:`force_cpu` is False, 
+            the output tensor will be stored in running device memory. The defalut value is None, the output tensor will stored in the CPU memory.
```
Comment (Boyan-Liu) [inline python/paddle/fluid/layers/control_flow.py]: 语法错误同上

File: python/paddle/fluid/layers/control_flow.py [hunk 5]
```diff
@@ -1530,27 +1558,41 @@ def not_equal(x, y, cond=None):
         cond(Variable, optional): If is :attr:`None`, the op will create a variable as output tensor, the shape and data type of this \
              tensor is the same as input :attr:`x`. If is not :attr:`None`, the op will set the variable as output tensor, the shape and data \
              type of this tensor should be the same as input :attr:`x`. Default value is :attr:`None`.
-
+        force_cpu(bool, optional): Whether force to store the output tensor in CPU. If :attr:`force_cpu` is False, 
+            the output tensor will be stored in running device memory. The defalut value is None, the output tensor will stored in the CPU memory.
```
Comment (Boyan-Liu) [inline python/paddle/fluid/layers/control_flow.py]: 语法错误同上

File: python/paddle/fluid/layers/control_flow.py (hunk 4 above)
Comment (Channingss) [reply python/paddle/fluid/layers/control_flow.py]: 已修改

**Pull request closed without merging**